audbcards/core/datacard.py

```diff
@@ -149,6 +151,12 @@ def file_duration_distribution(self) -> str:
         if len(durations) > 0:
             min_ = np.min(durations)
             max_ = np.max(durations)
+
+        # Skip creating a distribution plot,
+        # if all durations are the same
+        if min_ == max_:
```

Comment:
I understand that you are using the trailing underscore to avoid python keywords. But are min and max python keywords in the sense that e.g. `raise` would be? For me the use of the underscore makes intuitive sense - this is only  a question.

Reply:
Yes, `min` and `max` are build in functions. At the moment we don't use them, but to avoid getting an error when using them at a later stage in the same function, I thought it would be better to not overwrite them.

Reply:
Ok builtins but not keywords in [this sense](https://www.w3schools.com/python/python_ref_keywords.asp). As said I find this also intuitive. Therefore this is resolved.